languages/rust/reference/exercise-concepts/leap.md

```diff
@@ -0,0 +1,39 @@
+# Concepts of Leap
+
+I prefer the first example solution because its terseness follows what I've seen of idiosyncratic Rust. It also avoids the closure concept addition in the canonical solution.
```

Comment:
```suggestion
I prefer the first example solution because its terseness follows what I've seen of idiomatic Rust. It also avoids the closure concept addition in the canonical solution.
```

Idiosyncratic focuses on what is distinctive; a potential synonym is "quirky".
Idiomatic focuses on what is standard or best-practice; a potential synonym is "conventional".

Reply:
yes, exactly what I meant to say @coriolinus !